nifi-nar-bundles/nifi-amqp-bundle/nifi-amqp-processors/src/main/java/org/apache/nifi/amqp/processors/AbstractAMQPProcessor.java

```diff
@@ -84,6 +89,72 @@
             .allowableValues("0.9.1")
             .defaultValue("0.9.1")
             .build();
+    public static final PropertyDescriptor USE_SSL_PROTOCOL = new PropertyDescriptor.Builder()
+            .name("Use SSL protocol")
+            .description("Indicates whether or not SSL protocol should be used")
+            .allowableValues("true", "false")
+            .defaultValue("false")
+            .required(true)
+            .build();
+    public static final PropertyDescriptor VALIDATE_CERTIFICATES = new PropertyDescriptor.Builder()
```

Comment:
I am not sure why we would allow certificates _not_ be validated on a secure connection. I recommend removing this option, and I strongly oppose to the default value being `false` if it is allowed to remain.